app/views/groups/new.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="container vh-100 d-flex flex-column">
+<div class="d-flex justify-content-around fixed-top pt-2 align-items-center heading-actions">
+<%= link_to groups_path do %>
+  <span class="arrow"><i class="uil uil-arrow-left icon"></i></span>
+<% end %>
+<h1>Add Category</h1>
+<i class="uil uil-search"></i>
+</div>
+
+<%= form_for(@group) do |f| %>
+  <div class="m-2">
+  <%= f.label :name %>
+  <%= f.text_field :name, autofocus: true, autocomplete: "name", class: "form-control", placeholder: "Enter category name" %>
+</div>
+
+<div class="m-2">
+
+  <%= f.label :icon %>
+  <%= f.text_field :icon, autofocus: true, autocomplete: "icon", class: "form-control", placeholder: "Enter icon url"  %>
+</div>
+  <%= f.submit 'Create Category', class: "btn btn-primary add-new m-2" %>
+<% end %>
+</div>
```

Comment:
- [ ] [REQUIRED] Just as I have recommended on the transaction page, kindly implement the same for the category page as well. Let's ensure that the button colour is adjusted and the button position is fixed beneath the screen as it is in the provided design template.

- [ ] [REQUIRED] You have done a great job so far @ProgramKingAbel , however, it is a requirement to abide with the design template  provided in terms of colour,and the general layout. Let's make sure that the button to add transactions is green and fixed at the bottom of the screen just as it it in the design template provided.

Your Implementation | Template Design
:--------------------: | :-------------------:
<img width="307" alt="Screenshot 2023-09-29 at 22 26 32" src="https://github.com/ProgramKingAbel/E-nstaPay/assets/58692590/1c33bcac-efc6-4819-857f-2bcba85e54b5"> | <img width="405" alt="Screenshot 2023-09-29 at 22 05 34" src="https://github.com/ProgramKingAbel/E-nstaPay/assets/58692590/4ebbf360-a5cd-4aee-b665-68939fc917c8">